suave/builder/api/api_server.go

```diff
@@ -6,25 +6,25 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
-// sessionManager is the backend that manages the session state of the builder API.
-type sessionManager interface {
-	NewSession() (string, error)
+// SessionManager is the backend that manages the session state of the builder API.
```

Comment:
I noticed that this was an unexported type being returned from the public function `NewServer`, which is kinda awkward.  Exporting `SessionManager` is a drive-by improvement.